spec/lib/vmdb/loggers_spec.rb

```diff
@@ -0,0 +1,52 @@
+describe Vmdb::Loggers do
+  let(:log_file) { Rails.root.join("log", "foo.log").to_s }
+  describe "#create_multicast_logger" do
+    it "defaults the lower level loggers to `DEBUG`" do
+      log = described_class.send(:create_multicast_logger, log_file)
+
+      expect(log.loggers.first.level).to eq(Logger::DEBUG)
+      expect(log.loggers.to_a.size).to   eq(1)
+    end
+
+    context "in a container environment" do
+      around(:each) { |example| in_container_env(example) }
+
+      it "sets logger_instance and $container_log to debug" do
+        log = described_class.send(:create_multicast_logger, log_file)
+
+        expect(log.loggers.first.level).to     eq(Logger::DEBUG)
+        expect(log.loggers.to_a.last.level).to eq(Logger::DEBUG)
+      end
+    end
+  end
+  describe "#apply_config_value" do
+    it "will update the main lower level logger instance" do
+      log = described_class.send(:create_multicast_logger, log_file)
+      described_class.send(:apply_config_value, {:foo => :info}, log, :foo)
+
+      expect(log.loggers.first.level).to eq(Logger::INFO)
+      expect(log.loggers.to_a.size).to   eq(1)
+    end
+
+    context "in a container environment" do
+      around(:each) { |example| in_container_env(example) }
+
+      it "will always keep $container_log as DEBUG" do
+        log = described_class.send(:create_multicast_logger, log_file)
+        described_class.send(:apply_config_value, {:foo => :info}, log, :foo)
+
+        expect(log.loggers.first.level).to     eq(Logger::INFO)
+        expect(log.loggers.to_a.last.level).to eq(Logger::DEBUG)
+      end
+    end
+  end
+end
+
+def in_container_env(example)
+  old_env = ENV.delete('CONTAINER')
+  ENV['CONTAINER'] = '1'
```

Comment:
Ususally this is set to `"true"`

Reply:
The code has it as effectively a `nil` check, but I can change it if you would prefer.  I have always kept the practice when calling other toggle-able `ENV` vars has been `TOGGLE_SOMETHING=1`, but that is just my preference.

Reply:
Right, we're really just checking to see if the ENV variable is set or unset.  But to avoid confusion in case the value becomes important in the future, I'd prefer "true".
https://github.com/ManageIQ/manageiq-pods/blob/master/images/miq-app/Dockerfile#L9